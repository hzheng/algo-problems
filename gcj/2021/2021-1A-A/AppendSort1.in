100
2
30 16
2
24 16
3
47 25 34
2
15 40
3
29 96 98
2
42 24
3
5 50 7
2
28 31
3
3 2 1
3
100 100 100
2
18 26
3
46 82 69
3
86 47 31
2
3 75
2
72 72
3
1 1 1
2
15 89
3
31 2 62
3
29 12 92
3
74 76 88
2
14 97
3
57 64 22
2
65 33
2
10 19
3
70 65 56
2
36 13
2
63 62
3
57 10 80
2
94 43
3
31 40 69
3
24 65 87
3
1 41 49
3
1 2 3
3
88 8 8
2
52 8
3
21 86 80
3
89 8 8
2
33 9
3
4 20 87
2
12 40
2
39 41
3
8 86 92
2
80 72
2
82 65
3
85 91 18
3
36 89 44
3
41 9 76
3
65 46 55
3
80 67 47
3
11 80 68
3
22 27 76
2
66 13
2
90 19
3
80 46 43
2
42 27
3
97 61 17
2
76 82
3
72 68 88
3
75 83 80
3
40 63 35
3
35 82 28
2
33 11
2
25 24
3
99 99 99
2
85 68
2
100 68
3
72 90 69
3
67 73 71
2
39 45
2
35 52
3
78 36 68
2
21 54
3
26 9 98
2
91 58
3
69 35 70
3
21 79 51
3
35 21 3
3
61 99 66
3
38 38 48
3
9 9 91
2
7 37
2
13 2
2
18 58
3
49 78 50
2
91 38
3
69 86 37
3
36 78 3
3
85 45 78
2
24 79
3
99 98 97
3
58 99 52
3
90 46 28
2
90 29
3
36 18 83
2
37 11
3
8 74 78
3
10 74 42
3
27 32 68
3
59 93 20
3
71 49 6
