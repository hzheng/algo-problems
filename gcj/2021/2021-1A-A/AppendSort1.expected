Case #1: 1
Case #2: 1
Case #3: 2
Case #4: 0
Case #5: 0
Case #6: 1
Case #7: 1
Case #8: 0
Case #9: 3
Case #10: 2
Case #11: 0
Case #12: 1
Case #13: 3
Case #14: 0
Case #15: 1
Case #16: 2
Case #17: 0
Case #18: 3
Case #19: 2
Case #20: 0
Case #21: 0
Case #22: 1
Case #23: 1
Case #24: 0
Case #25: 3
Case #26: 1
Case #27: 1
Case #28: 2
Case #29: 1
Case #30: 0
Case #31: 0
Case #32: 0
Case #33: 0
Case #34: 3
Case #35: 1
Case #36: 1
Case #37: 4
Case #38: 1
Case #39: 0
Case #40: 0
Case #41: 0
Case #42: 0
Case #43: 1
Case #44: 1
Case #45: 1
Case #46: 1
Case #47: 2
Case #48: 2
Case #49: 3
Case #50: 1
Case #51: 0
Case #52: 1
Case #53: 1
Case #54: 3
Case #55: 1
Case #56: 3
Case #57: 0
Case #58: 2
Case #59: 1
Case #60: 1
Case #61: 1
Case #62: 1
Case #63: 1
Case #64: 2
Case #65: 1
Case #66: 1
Case #67: 1
Case #68: 1
Case #69: 0
Case #70: 0
Case #71: 2
Case #72: 0
Case #73: 1
Case #74: 1
Case #75: 2
Case #76: 1
Case #77: 3
Case #78: 1
Case #79: 2
Case #80: 1
Case #81: 0
Case #82: 1
Case #83: 0
Case #84: 1
Case #85: 1
Case #86: 1
Case #87: 2
Case #88: 2
Case #89: 0
Case #90: 3
Case #91: 1
Case #92: 3
Case #93: 1
Case #94: 2
Case #95: 1
Case #96: 0
Case #97: 1
Case #98: 0
Case #99: 1
Case #100: 3
